### PR TITLE
Correct the courseware imports

### PR DESCRIPTION
**Pull request by nedbat**
#### What are the relevant tickets?
(None)

#### What's this PR do?
edx-platform has long warned that "courseware" should be imported as "lms.djangoapps.courseware".  This PR makes that correction.

#### How should this be manually tested?
Unsure.

### CODE REVIEW

**Comment by pdpinch** [comment]
@tasawernawaz can you review this?

**Comment by nedbat** [comment]
@pdpinch Oops, I should have made this a draft.  It's part of a larger fix across repos.

**Comment by tasawernawaz** [comment]
> @pdpinch Oops, I should have made this a draft. It's part of a larger fix across repos.

It means this isn't ready for review yet, right ?

**Comment by pdpinch** [comment]
@tasawernawaz you can hold off, but please keep it in mind while you're working on #252 -- it would be convenient to address these both in one release.

**Comment by nedbat** [comment]
I'm hoping that this change is safe to make here even without any changes in edx-platform.  But to be honest, I don't understand all the forces at work here.

**Comment by nedbat** [comment]
@pdpinch @tasawernawaz ?

File: RELEASE.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Release Notes
 =============
 
+Version 0.8.4
+-------------
+
+- Correct some imports
+
```
Comment (pdpinch) [inline RELEASE.rst]: Can you revert the change to this file? We have tooling that creates the release notes, and we're likely to roll this change up with copy/paste bugfix, #254

File: edx_sga/__init__.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = '0.8.3'
+__version__ = '0.8.4'
```
Comment (pdpinch) [inline edx_sga/__init__.py]: Can you revert the change to this file? We have tooling that handles version numbers, and we're likely to roll this change up with copy/paste bugfix, #254

Comment (nedbat) [reply edx_sga/__init__.py]: Done

File: RELEASE.rst (hunk 1 above)
Comment (nedbat) [reply RELEASE.rst]: Done

**Comment by tasawernawaz** [comment]
@nedbat I installed your branch on my local devstack and while getting course with SGA problem already added I am getting following error.
```
edx.devstack.studio  | Traceback (most recent call last):
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/xblock/plugin.py", line 146, in load_classes
edx.devstack.studio  |     yield (class_.name, cls._load_class_entry_point(class_))
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/xblock/plugin.py", line 75, in _load_class_entry_point
edx.devstack.studio  |     class_ = entry_point.load()
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2434, in load
edx.devstack.studio  |     return self.resolve()
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2440, in resolve
edx.devstack.studio  |     module = __import__(self.module_name, fromlist=['__name__'], level=0)
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/src/edx-sga/edx_sga/sga.py", line 16, in <module>
edx.devstack.studio  |     from lms.djangoapps.courseware.models import StudentModule  # lint-amnesty, pylint: disable=import-error
edx.devstack.studio  |   File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/models.py", line 74, in <module>
edx.devstack.studio  |     class StudentModule(models.Model):
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/base.py", line 319, in __new__
edx.devstack.studio  |     new_class._meta.apps.register_model(new_class._meta.app_label, new_class)
edx.devstack.studio  |   File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/apps/registry.py", line 224, in register_model
edx.devstack.studio  |     (model_name, app_label, app_models[model_name], model))
edx.devstack.studio  | RuntimeError: Conflicting 'studentmodule' models in application 'courseware': <class 'courseware.models.StudentModule'> and <class 'lms.djangoapps.courseware.models.StudentModule'>.
```

**Comment by nedbat** [comment]
Yes, these are the same errors showing on Travis.  This change needs the corresponding changes in edx-platform: https://github.com/edx/edx-platform/pull/21581.  That pull request is ready to merge, so we should be able to make both changes at the same time.

**Comment by nedbat** [comment]
@tasawernawaz @pdpinch I don't have the rights to merge in this repo... :(

**Comment by nedbat** [comment]
Thanks! I've updated the pull request in edx-platform to use this new merged code. But I guess it should wait for an official tagged version here?
